polaris-service/src/test/java/org/apache/polaris/service/PolarisApplicationIntegrationTest.java

```diff
@@ -179,6 +203,14 @@ public static void deletePrincipalRole() {
         .close();
   }
 
+  private static void assertZeroErrorsInApplicationLog() {
+    assertThat(new File(CURRENT_LOG.get()))
+        .exists()
+        .content()
+        .hasSizeGreaterThan(0)
+        .doesNotContain("ERROR", "FATAL", "Exception");
```

Comment:
I think a warn-level log could also have `Exception`, which shouldn't be considered as an error.

Reply:
Fixed.